[INFO] 2025-07-21 21:31:14,415 django.utils.autoreload - Watching for file changes with StatReloader
[WARNING] 2025-07-21 21:31:21,293 django.request - Unauthorized: /
[WARNING] 2025-07-21 21:31:21,294 django.server - "GET / HTTP/1.1" 401 4989
[INFO] 2025-07-21 21:31:22,146 django.server - "GET /static/rest_framework/css/bootstrap.min.css HTTP/1.1" 200 121457
[INFO] 2025-07-21 21:31:22,157 django.server - "GET /static/rest_framework/css/bootstrap-tweaks.css HTTP/1.1" 200 3426
[INFO] 2025-07-21 21:31:22,234 django.server - "GET /static/rest_framework/js/csrf.js HTTP/1.1" 200 1793
[INFO] 2025-07-21 21:31:22,284 django.server - "GET /static/rest_framework/css/prettify.css HTTP/1.1" 200 817
[INFO] 2025-07-21 21:31:22,297 django.server - "GET /static/rest_framework/css/default.css HTTP/1.1" 200 1152
[INFO] 2025-07-21 21:31:22,297 django.server - "GET /static/rest_framework/js/ajax-form.js HTTP/1.1" 200 3796
[INFO] 2025-07-21 21:31:22,330 django.server - "GET /static/rest_framework/js/bootstrap.min.js HTTP/1.1" 200 39680
[INFO] 2025-07-21 21:31:22,357 django.server - "GET /static/rest_framework/js/load-ajax-form.js HTTP/1.1" 200 59
[INFO] 2025-07-21 21:31:22,370 django.server - "GET /static/rest_framework/js/prettify-min.js HTTP/1.1" 200 13632
[INFO] 2025-07-21 21:31:22,394 django.server - "GET /static/rest_framework/js/default.js HTTP/1.1" 200 1268
[INFO] 2025-07-21 21:31:22,427 django.server - "GET /static/rest_framework/img/grid.png HTTP/1.1" 200 1458
[INFO] 2025-07-21 21:31:22,525 django.server - "GET /static/rest_framework/js/jquery-3.7.1.min.js HTTP/1.1" 200 87533
[INFO] 2025-07-21 21:31:41,464 django.server - "GET /docs HTTP/1.1" 301 0
[ERROR] 2025-07-21 21:31:42,017 django.request - Internal Server Error: /docs/
Traceback (most recent call last):
  File "D:\nutrition\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\nutrition\.venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "D:\nutrition\.venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "D:\nutrition\.venv\Lib\site-packages\rest_framework\response.py", line 74, in rendered_content
    ret = renderer.render(self.data, accepted_media_type, context)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\nutrition\.venv\Lib\site-packages\rest_framework\renderers.py", line 160, in render
    template = self.resolve_template(template_names)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\nutrition\.venv\Lib\site-packages\rest_framework\renderers.py", line 170, in resolve_template
    return loader.select_template(template_names)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\nutrition\.venv\Lib\site-packages\django\template\loader.py", line 47, in select_template
    raise TemplateDoesNotExist(", ".join(template_name_list), chain=chain)
django.template.exceptions.TemplateDoesNotExist: drf_spectacular/swagger_ui.html
[ERROR] 2025-07-21 21:31:42,030 django.server - "GET /docs/ HTTP/1.1" 500 91396
[ERROR] 2025-07-21 21:31:46,168 django.request - Internal Server Error: /docs/
Traceback (most recent call last):
  File "D:\nutrition\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\nutrition\.venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "D:\nutrition\.venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "D:\nutrition\.venv\Lib\site-packages\rest_framework\response.py", line 74, in rendered_content
    ret = renderer.render(self.data, accepted_media_type, context)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\nutrition\.venv\Lib\site-packages\rest_framework\renderers.py", line 160, in render
    template = self.resolve_template(template_names)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\nutrition\.venv\Lib\site-packages\rest_framework\renderers.py", line 170, in resolve_template
    return loader.select_template(template_names)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\nutrition\.venv\Lib\site-packages\django\template\loader.py", line 47, in select_template
    raise TemplateDoesNotExist(", ".join(template_name_list), chain=chain)
django.template.exceptions.TemplateDoesNotExist: drf_spectacular/swagger_ui.html
[ERROR] 2025-07-21 21:31:46,171 django.server - "GET /docs/ HTTP/1.1" 500 91670
[WARNING] 2025-07-21 21:31:50,341 django.request - Not Found: /user
[WARNING] 2025-07-21 21:31:50,342 django.server - "GET /user HTTP/1.1" 404 16986
[WARNING] 2025-07-21 21:32:07,911 django.request - Method Not Allowed: /auth/login/
[WARNING] 2025-07-21 21:32:07,914 django.server - "GET /auth/login/ HTTP/1.1" 405 7707
